pyproject.toml

```diff
@@ -131,7 +131,85 @@ exclude = [
 ]
 
 [tool.mypy]
-files = ["sphinx", "utils"]
+files = ["sphinx", "utils", "tests"]
+exclude = [
```

Comment:
is this easier than using the `[[tool.mypy.overrides]]` syntax?

Reply:
I don't know how to *disable* everything when using the overrides syntax. With the overrides, I'm not sure you can say "ignore this module".

Reply:
you can glob using that syntax, as in:

```toml
[[tool.mypy.overrides]]
module = [
    "tests.module.*",
]
ignore_errors = true
```

which also allows you to suppress only specific error codes.
Also i'm not sure if ignoring files entirely just suppresses errors, or also prevents mypy from using the type information in those files to lint other files? dunno

Reply:
I totally forgot the ignore_errors flag (I never use it in general).

Mmmh, technically there shouldn't be any dependency between files although I remember one somewhere when I was playing with intersphinx. But for that, I'd say we should make a dedicated util module then.

The reason why I picked the regex is because I can avoid redundant prefixes, but sure, I can do the overrides way.

Reply:
Ok, it's cleaner without the regex and at least it passes through every file (and probably picks up the lint of whatever it could deduce).